ambry-router/src/main/java/com.github.ambry.router/DeleteManager.java

```diff
@@ -102,6 +106,7 @@ void submitDeleteBlobOperation(String blobIdString, FutureResult<Void> futureRes
       DeleteOperation deleteOperation =
           new DeleteOperation(routerConfig, routerMetrics, responseHandler, blobId, futureResult, callback, time);
       deleteOperations.add(deleteOperation);
+      readyForPollCallback.onPollReady();
```

Comment:
not sure why we need this. Why not just call this from the `OperationController`?